ledger-service/http-json-perf/daml/LargeAcs.daml

```diff
@@ -114,3 +114,18 @@ template NotIou
 
 data WhichTemplate = UseIou | UseNotIou
   deriving (Eq, Show)
+
+template KeyedIou
+  with
+    id: Int
+    issuer : Party
+    owner : Party
+    currency : Text
+    amount : Decimal
+    observers : [Party]
+  where
+    signatory issuer, owner
+    observer observers
+
+    key (issuer, id): (Party, Int)
+    maintainer key._1
```

Comment:
newline missing at the end of the file, can you add it?

Reply:
I guess my use of mergify wasn't the best here , I should have used that tag after the initial review. I will make this change in my followup PR's